README.md

```diff
@@ -18,13 +18,7 @@ The FTP is implemented by Pilz GmbH & Co. KG.
 Relevant changes are cherry-picked into `kinetic-devel` on a case-by-case basis.
 
 ## Package: pilz_trajectory_generation
-Provides the generators to create LIN, PTP and CIRC trajectories. All of these commands can be combined and
-executed in sequence, with and without blending, using a dedicated MoveIt! capability.
-
-Custom commands can easily be defined and integrated due to the usage of a plugin structure.
-
-## Package: pilz_extensions
-Minor extensions of existing packages needed for the trajectory_generation.
+This package used to be found here but is now moved to [moveit](https://moveit.ros.org/documentation/planners/
```

Comment:
```suggestion
This package used to be found here but is now moved to [moveit](https://moveit.ros.org/documentation/planners/)
```

Reply:
A similar note should be added to [ROS wiki](http://wiki.ros.org/pilz_trajectory_generation) - probably during the follow-up PBI ROS-849?